hudi-common/src/main/avro/HoodieCleanerPlan.avsc

```diff
@@ -47,6 +47,7 @@
       "type": "string"
     },
     {
+      /** This is deprecated and replaced by the field filePathsToBeDeletedPerPartition **/
```

Comment:
I am adding a new field filePathsToBeDeletedPerPartition which deprecates filesToBeDeletedPerPartition field. filePathsToBeDeletedPerPartition retains information to distinguish cleaning of regular hudi and bootstrap base files.